src/com/lilithsthrone/game/sex/sexActions/submission/SALyssiethSpecials.java

```diff
@@ -132,6 +132,10 @@ public void applyEffects() {
 			Main.game.getPlayer().setBodySize(BodySize.TWO_AVERAGE.getMedianValue());
 			Main.game.getPlayer().setHeight(168);
 			Main.game.getPlayer().setAssType(AssType.DEMON_COMMON);
+			if(Main.game.getPlayer().getBreastCrotchType()!=BreastType.NONE) {
+				Main.game.getPlayer().setBreastCrotchType(BreastType.DEMON_COMMON);
+				// Main.game.getPlayer().setBreastCrotchSize(CupSize.D); // leave as is?
```

Comment:
Please tell me, if this should be removed aka leave the crotch cup size as is or to be set to a certain value. Personally I'd prefer the former.